src/dynarec/rv64/dynarec_rv64_0f.c

```diff
@@ -344,6 +344,20 @@ uintptr_t dynarec64_0F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
             }
             break;
 
+        case 0x5A:
+            INST_NAME("CVTPS2PD Gx, Ex");
+            nextop = F8;
+            GETGX(x1);
+            GETEX(x2, 0);
+            s0 = fpu_get_scratch(dyn);
+            FLW(s0, wback, fixedaddress);
```

Comment:
both Read should be done at the same time, in case Gx==Ex, to avoid overwrite PS[1] with PD[0]